workflow/rocoto/gefs_tasks.py

```diff
@@ -9,48 +10,63 @@ def __init__(self, app_config: AppConfig, cdump: str) -> None:
         super().__init__(app_config, cdump)
 
     def stage_ic(self):
-
         cpl_ic = self._configs['stage_ic']
-
         deps = []
-
+        if self._configs['base']['END_OF_IAU_START']:
+            DTG = self._configs['base']['SDATE'] + timedelta(hours=3)
+            DTG_PREFIX = DTG.strftime('%Y%m%d') + '.' + DTG.strftime('%H') + '0000'
+        else:
+            DTG_PREFIX = "@Y@m@d.@H0000"
```

Comment:
This would fail if used on an experiment of more than one cycle. Note how the non-replay case is using the rocoto `<cyclestr>` formatting (`@Y`, `@m`, etc.). The replay needs to do the same instead of using `SDATE` (see also next comment).
```suggestion
        dtg_prefix = "@Y@m@d.@H0000"
        if self._configs['base']['END_OF_IAU_START']:
            offset = "+03:00:00"
        else:
            offset = None
```